packages/suite/src/components/wallet/CoinmarketBuyButton/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import * as routerActions from 'src/actions/suite/routerActions';
+import { Button } from '@trezor/components';
+import { NetworkSymbol } from 'src/types/wallet';
+import { Translation } from 'src/components/suite';
+import { useActions, useAccountSearch } from 'src/hooks/suite';
+
+interface BuyButtonProps {
+    symbol: NetworkSymbol;
+    dataTest: string;
+}
+
+const CoinmarketBuyButton = ({ symbol, dataTest }: BuyButtonProps) => {
+    const { goto } = useActions({ goto: routerActions.goto });
```

Comment:
The preferred way to execute actions is to `import { useDispatch } from 'src/hooks/suite';` and `const dispatch = useDispatch();`. We should mark `useActions` as deprecated.